.github/scripts/gemini-translator.py

```
@@ -0,0 +1,515 @@
+    success_count = len(results.successful_languages)
+    if success_count > 0:
+        languages_joined = ','.join(results.successful_languages)
+        with open(os.environ['GITHUB_OUTPUT'], 'a') as f:
+            f.write(f"languages-updated={languages_joined}\n")
+            f.write(f"translation-cost=${results.total_cost:.4f}\n")
+            f.write(f"strings-translated={results.total_strings}\n")
+            f.write(f"processing-time={processing_time:.2f}\n")
+            f.write(f"api-calls={cost_summary['api_calls']}\n")
+            f.write(f"tokens-used={cost_summary['tokens_used']}\n")
+            f.write(f"remaining-budget={cost_summary['remaining_budget']:.2f}\n")
+            f.write(f"translation-success=true\n")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)